ABC.java

```diff
@@ -0,0 +1,5 @@
+
+class Hello
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/app/gagsdeep/TestSonar/pullRequest?prid=4663122)